examples/starter/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "public_key" {
+  description = "Public API key to authenticate to Atlas"
+}
+variable "private_key" {
+  description = "Private API key to authenticate to Atlas"
+}
+variable "dbuser" {
+  description = "MongoDB Atlas Database User"
+}
+variable "" { //Declare and initialize variable for the db-user-password and set the password accordingly
```

Comment:
@nikhil-mongo  I'm so sorry but I read this too quickly and was thinking it was an example .tfvars file!!   This suggestion must have made no sense to you :)   Some days I try to do too many things at once.  Please set back to dbuser_password and all the other changes are nicely made.  Thank you :D

Reply:
made changes to the variables.tf and Readme.md.